text_formatter.go

```diff
@@ -68,8 +68,6 @@ func (f *TextFormatter) Format(entry *Entry) ([]byte, error) {
 
 	b := &bytes.Buffer{}
 
-	prefixFieldClashes(entry.Data)
```

Comment:
Should this be `prefixFieldClashes(data, entry.Logger.showCaller, entry.depth)` to match the change in `JSONFormatter`?